### ISSUE TITLE
make install fails

### ISSUE THREAD

**Issue by haansn08** [math-comp/multinomials#75]
The manual install as specified in the README does not work:
```
$ make install
make: *** No rule to make target 'install'.  Stop.
```
`dune` also does nothing unless one sets some parameters (?, man page of dune and dune documentation say nothing about `mandir`)
```
$ dune install
Error: The mandir installation directory is unknown.
Hint: It could be specified with --mandir
```

I have set my installation path via `$COQLIBINSTALL=/home/user/.local/share/coq`.

**Comment by strub** [comment]
I have this:

```
% dune install
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq-mathcomp-multinomials/META
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq-mathcomp-multinomials/dune-package
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq-mathcomp-multinomials/opam
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/freeg.v
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/freeg.vo
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/monalg.v
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/monalg.vo
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/mpoly.v
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/mpoly.vo
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/ssrcomplements.v
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/ssrcomplements.vo
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/xfinmap.v
Installing /Users/strubpy/Projects/perso/multinomials/_opam/lib/coq/user-contrib/SsrMultinomials/xfinmap.vo
Installing /Users/strubpy/Projects/perso/multinomials/_opam/doc/coq-mathcomp-multinomials/LICENSE
Installing /Users/strubpy/Projects/perso/multinomials/_opam/doc/coq-mathcomp-multinomials/README.md
```

Please, provide your version of `dune`

**Comment by haansn08** [comment]
```
$ dune --version
n/a

$ pacman -Qi dune
Name            : dune
Version         : 3.6.2-1
Description     : A composable build system for OCaml (formerly jbuilder)
...
```
Since opam is mentioned in your install log, maybe it is required for dune?

**Comment by strub** [comment]
That `dune` answers `n/a` is pretty surprising. For instance, in my case, if I take a non opam-installed version of dune:

```
/opt/homebrew/bin/dune --version
3.7.0
```

IMO, there is something fishy with your installation

**Comment by haansn08** [comment]
Maybe there is a problem with the Arch package, but this seems unlikely. I can use my installation to build the master branch of coq as of today.

**Comment by haansn08** [comment]
I just upgraded to `dune 3.7.0.post1-1` and the issue persists.

**Comment by Alizter** [comment]
If you are doing `dune install` and it is complaining about `--mandir` then (confusingly) this is because Dune was unable to guess the install prefix. When used in conjunction with opam, Dune is able to guess the prefix easily, but when `.opam` isn't around it can no longer rely on that heuristic.

To fix this, pass the install prefix with `dune install --prefix /path/to/install`. The unhelpful error message has been fixed in newer versions of Dune.

- https://github.com/ocaml/dune/issues/7814
- https://github.com/ocaml/dune/pull/7844

The fix appeared in Dune 3.9.
